### PR TITLE
Update [...2]onboard.js-migration-guide.md

### PR DESCRIPTION
**Pull request by laurencedelisle**
### Description
Added in missing links and updated some punctuation/grammar for docs.

### Checklist
- [x] The version field in `package.json` is incremented following [semantic versioning](https://semver.org/)
- [x] The box that allows repo maintainers to update this PR is checked
- [x] I tested locally to make sure this feature/fix works
- [x] I have run `yarn check` & `yarn build` to confirm there are not any associated errors
- [x] This PR passes the Circle CI checks

### CODE REVIEW

File: docs/src/routes/docs/[...1]overview/[...2]onboard.js-migration-guide.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ const onboard = Onboard({
 })
 ```
 
-You can find the full list of initialization options in our docs here.
+You can find the full list of initialization options in our docs [here](https://onboard.blocknative.com/docs/packages/core#options).
```
Comment (laurencedelisle) [inline docs/src/routes/docs/[...1]overview/[...2]onboard.js-migration-guide.md:49]: Should this be a relative link or absolute?

**Comment by laurencedelisle** [comment]
Please double check that the links are correct!

File: docs/src/routes/docs/[...1]overview/[...2]onboard.js-migration-guide.md (hunk 1 above)
Comment (Adamj1232) [reply docs/src/routes/docs/[...1]overview/[...2]onboard.js-migration-guide.md:49]: As is will work great! 🤓

File: docs/src/routes/docs/[...1]overview/[...2]onboard.js-migration-guide.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ We've also added more expansive custom CSS properties so you can style every par
 }
 ```
 
-The full list of CSS variables for web3-onboard are available here.
+The full list of CSS variables for web3-onboard are available [here](https://onboard.blocknative.com/docs/packages/core#custom-styling).
```
Comment (Adamj1232) [inline docs/src/routes/docs/[...1]overview/[...2]onboard.js-migration-guide.md:92]: You could add a link to our custom styling tool - https://onboard.blocknative.com/theming-tool
Its shown as an experimental option [here](https://onboard.blocknative.com/docs/getting-started/customization) as an example not that it needs to be copied

File: docs/src/routes/docs/[...1]overview/[...2]onboard.js-migration-guide.md (hunk 1 above)
Comment (gesquinca) [reply docs/src/routes/docs/[...1]overview/[...2]onboard.js-migration-guide.md:49]: We should use relative links whenever possible. You can read about them in the kit docs documentation here.

https://kit-docs.svelteness.dev/docs/markdown/extensions#links

Comment (laurencedelisle) [reply docs/src/routes/docs/[...1]overview/[...2]onboard.js-migration-guide.md:49]: @gesquinca sounds good. All the docs currently use absolute so I can try and go in and fix those as well.

**Review by Adamj1232 — APPROVED** [review]

**Pull request merged**